npm/css-src/sass/main.scss

```diff
@@ -1,6 +1,44 @@
 @import "node_modules/govuk-frontend/govuk/all";
 @import "overrides/all";
 
+/* Variables for repeated blocks */
+$borderRadius: 10px;
+$boxShadow: 0 2px 3px govuk-colour("black", $legacy: "black") inset;
+$position: relative;
+
+.progress-display[value] {
+    /* Get rid of the default appearance */
+    appearance: none;
+    border: none;
+    /* Dimensions */
+    width: 100%;
+    height: 20px;
+    background-color: govuk-colour("light-grey", $legacy: "grey-1");
+    border-radius: $borderRadius;
+    box-shadow: $boxShadow;
+    color: govuk-colour("blue", $legacy: "govuk-blue");
+    position: $position;
+    margin: 0 0 1.5em;
```

Comment:
Just noticed - this size should be in px for consistency with the other ones (and with the GOV.UK design system, which [seems to use px](https://github.com/alphagov/govuk-frontend/blob/v3.6.0/src/govuk/settings/_measurements.scss#L16-L16) too).

Reply:
No worries, that's now been updated!